### PR TITLE
Update populate-a-dictionary-exercise.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Replace the zip method with `fromkeys` in the `populate a dictionary` python exercise

Reason:
- The `zip` method isn't taught until the functional-programming course

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/python/python-core/populate-a-dictionary-exercise/populate-a-dictionary-exercise.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/python/python-core/populate-a-dictionary-exercise/populate-a-dictionary-exercise.md?ref=4bb27887563e4c584b69c6d15816a30f472528a1)

File: python/python-core/populate-a-dictionary-exercise/populate-a-dictionary-exercise.md [hunk 1]
```diff
@@ -27,17 +27,15 @@ setupCode:
...
-keys = ['Name', 'Surname', 'Profession']
-values = ["Alan", "Turing", "Mathematician"]
-```
-
-Don't forget to output the new `dictionary`.
+> 👩‍💻 Your task is to:
+> - **create a dictionary with at least one key-value pair**
+> - **save the values in a different variable**
+> - **save the keys in a different variable**
+> - **clear the dictionary**
```
Comment (kapnobatai137) [inline python/python-core/populate-a-dictionary-exercise/populate-a-dictionary-exercise.md:34]: how could we make it clear that they should use dictionary methods for these three?

maybe something like this?

```suggestion
> 👩‍💻 Using dictionary methods, your task is to:
> - **create a dictionary with at least one key-value pair**
> - **save the values in a different variable**
> - **save the keys in a different variable**
> - **clear the dictionary**
```

Comment (Stefan-Stojanovic) [reply python/python-core/populate-a-dictionary-exercise/populate-a-dictionary-exercise.md:34]: Yeah, that sounds better

**Review by nem035 — APPROVED** [review]

**Pull request merged**